src/styles/blocks/card.css

```diff
@@ -0,0 +1,92 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+}
+
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 0 16px;
+  box-sizing: border-box;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px auto 40px;
+  display: flex;
```

Comment:
the flex display is not necessary for the '.card__image' class since it has only one child element. Remove the 'display: flex;' style. You can use either display: block or display: inline-block